diff --git a/3rdparty/tvm/include/tvm/ir/op.h b/3rdparty/tvm/include/tvm/ir/op.h
index 48cf61d18..17607cb53 100644
--- a/3rdparty/tvm/include/tvm/ir/op.h
+++ b/3rdparty/tvm/include/tvm/ir/op.h
@@ -146,7 +146,7 @@ class OpNode : public RelayExprNode {
   // Internal function to compute if it is primitive op
   bool IsPrimitiveOp_() const {
     const auto& fn_ty = this->op_type;
-    ICHECK(fn_ty.get() != nullptr);
+    if (fn_ty.get() == 0) return false;
     if (fn_ty->type_constraints.size() != 1) return false;
     const TypeRelationNode* rel = fn_ty->type_constraints[0].as<TypeRelationNode>();
     if (rel == nullptr) return false;
